tests/test_utils_rest.py

```
@@ -0,0 +1,60 @@
+# Copyright European Organization for Nuclear Research (CERN) since 2012
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+
+import pytest
+from werkzeug import exceptions
+
+from rucio.web.rest.flaskapi.v1.common import param_get_bool
+
+
+@pytest.mark.parametrize(
+    'params,kwargs,raise_log,raise_error',
+    [
+        ({"object": False}, {}, False, False),
```

Can we add extra cases, e.g.

```python
({"object": True}, {}, False, False),            # Pure True value - no warning
({}, {"default": False}, False, False),          # default=False works
({"object": '1'}, {}, True, False),             # String '1' - logs warning, returns True
({"object": None}, {"default": False}, False, False),  # Explicit None with default
```

Additional case variations:

```python
({"object": "TRUE"}, {}, False, False),
({"object": "False"}, {}, False, False), 
({"object": "FALSE"}, {}, False, False),
```

Cases to be cautious about (need manual testing first):

```python
({"object": 0}, {}, True, False)  # Integer 0 - MIGHT trigger warning due to int(0) succeeding
```

---

The explicit None should raise an error (set an invalid type parameter) but otherwise adding the additional tests